monai/bundle/scripts.py

```diff
@@ -217,10 +217,15 @@ def _remove_ngc_prefix(name: str, prefix: str = "monai_") -> str:
 
 
 def _download_from_ngc(
-    download_path: Path, filename: str, version: str, remove_prefix: str | None, progress: bool
+    download_path: Path,
+    filename: str,
+    version: str,
+    prefix: str = "monai_",
```

Comment:
Do we also need add this arg in the download and pass it to this one?

Reply:
Discussed this offline with @yiheng-wang-nv 
Will leave this one for this PR. If we have a case need this arg in the download and load, will enhance them later.